src/box/lua/config/applier/box_cfg.lua

```diff
@@ -510,6 +518,25 @@ local function post_apply(config)
     box.space._cluster:on_replace(names_cluster_on_replace)
     box.space._schema:on_replace(names_schema_on_replace)
 
+    local upgrade_on_replace = nil
+    local version = mkversion.get()
+    if version < mkversion(3, 0, 0) then
```

Comment:
@sergepetrenko, maybe we should not start this fiber, until the schema is below 3.0.0. Maybe setting on_replace triggers for _schema/_cluster and starting the fiber should be done only after upgrade? This way we won't have hanging fiber on < 3.0.0. What do you think?

Reply:
Yep, good idea.

Reply:
Done

Reply:
Nah, it was a bad idea. The problem there is that, on_replace triggers must be anyway set before upgrade is done. After configuring everything (on_replace triggers, watchers, fibers) after schema upgrade upgrade test is very flaky: we may receive some names before these triggers are set, alert is not deleted, box.cfg is not called in such case. Let's return to the old variant

Reply:
Applied in the new version.